### PR TITLE
Add execution-details and inbound-parse to node sdk

### LINKED ISSUE #4164: 🚀 Feature: missing methods in Node SDK
### 🔖 Feature description

Two methods are missing in the Node SDK:
- execution details: /v1/execution-details
- inbound parse: /v1/inbound-parse/mx/status

Please add these to the SDK

### 🎤 Why is this feature needed ?

To bring the SDK on par with the API endpoint methods available

### ✌️ How do you aim to achieve this?

Tagging @jainpawan21 to work on this issue, as requested by Prosper.

### 🔄️ Additional Information

_No response_

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

None

### PR DESCRIPTION
**Pull request by peoray**
### What change does this PR introduce?
Fixes #4164

### CODE REVIEW

**Comment by peoray** [comment]
@jainpawan21 please take a look

File: packages/node/src/lib/execution-details/execution-details.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { IExecutionDetails } from './execution-details.interface';
+import { WithHttp } from '../novu.interface';
+
+export class ExecutionDetails extends WithHttp implements IExecutionDetails {
+  async get() {
+    return await this.http.get(`/execution-details`);
```
Comment (jainpawan21) [inline packages/node/src/lib/execution-details/execution-details.ts:6]: You missed params notificationId and subscriberId for this https://api.novu.co/api#/Execution%20Details/ExecutionDetailsController_getExecutionDetailsForNotification

**Comment by jainpawan21** [comment]
add these methods into readme file as well

**Comment by jainpawan21** [comment]
@peoray 

Let me know if you need any help

File: packages/node/src/lib/execution-details/execution-details.ts (hunk 1 above)
Comment (peoray) [reply packages/node/src/lib/execution-details/execution-details.ts]: good spot, thanks.
fixed!

**Comment by peoray** [comment]
@jainpawan21 PR has been updated, please take a look :)

File: packages/node/README.md [hunk 2]
```diff
@@ -152,6 +152,13 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [Environments](#environments)
 - [Layouts](#layouts)
 - [Integrations](#integrations)
+- [Feeds](#feeds)
+- [Changes](#changes)
+- [Environments](#environments)
+- [Tenants](#tenants)
+- [Workflows](#workflows)
```
Comment (jainpawan21) [inline packages/node/README.md:159]:
```suggestion
```

File: packages/node/README.md [hunk 3]
```diff
@@ -1131,3 +1152,31 @@ await novu.notificationTemplates.getAll({
...
+
+```ts
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<NOVU_API_KEY>');
+
+const executionDetailsParams = {
+  page: 0, //optional
+  pageSize: 20,
+  key: "topicKey"
```
Comment (jainpawan21) [inline packages/node/README.md:1166]:
```suggestion
  subscriberId: 'subscriberId_123'
  notificationId: 'notificationid_abcd
```

**Review by jainpawan21 — CHANGES_REQUESTED** [review]

**Review by jainpawan21 — APPROVED** [review]

**Pull request merged**